src/modules/articles/components/RecommendedRow.js

```diff
@@ -89,6 +90,8 @@ const RecommendedRow = ({
     return null;
   }
 
+  console.log(data);
```

Comment:
Thanks for fixing this bug! I'll merge it in once you remove this print statement.